src/app/features/location/geo.service.ts

```diff
@@ -66,18 +66,35 @@ export class GeoService {
       );
   }
 
-  private reformatDisplayName(result: OpenStreetMapsSearchResult): GeoResult {
+  private getCity(addr: any): string {
```

Comment:
I think you can use the type `OpenStreetMapsSearchResult` instead of `any` for the addr to these methods?